site/docs/components/inline-notification.mdx

```diff
@@ -56,10 +92,6 @@ E.g. **Meaning changed** We prevent significant changes to the meaning of questi
 *   Dismissible or persistent:
     *   Use **Persistent** notifications when the information continues to be relevant after the message is shown, e.g. Defining the reporting size on an Insight report
     *   Use **Dismissible** notifications when the information has been acknowledged and the message is no longer relevant, e.g. helpfully updating formatting for user input data.
-*   Anatomy:
-    *   **Heading**: summarizes the message. This relates directly to the message. Do not use generic language such as “Success”, “Error”, etc.
-    *   **Description**: succinct description that is a standalone sentence (that is, it does not run on from the heading).
-    *   **Call to action** (optional): hyperlinks can be included to provide the user with a clear call to action.
```

Comment:
I can see how you've incorporated the copy guidance here into guidelines further up the page. It's useful to have the "anatomy" outlined so that we can refer to specific parts of the component consistently when talking to each other. Can you think of a way to outline the anatomy here without muddling it with copy guidelines?

Reply:
I've added in a section for anatomy but haven't gone into detail. Does this help resolve?

Reply:
Sure!